src/avatar-group.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import Avatar from './avatar'
+import Row from './row'
+import Column from './column'
+import Group from './group'
+
+const sizes = {
+  xs: [1],
+  sm: [3],
+  md: [5],
+  lg: [7],
+  xl: [9],
+}
+
+const AvatarGroup = ({
+  members,
+  direction = 'horizontal',
+  align,
+  spacing = 'md',
+  width,
+  maxWidth,
+  fixedCount,
+  sx,
+}) => {
+  if (members.length > fixedCount) {
+    throw Error(
+      `cannot render '${members.length}' avatars with a fixed count of '${fixedCount}'`
+    )
+  }
+
+  let gap
+  if (Object.keys(sizes).includes(spacing)) {
```

Comment:
another one of these
```suggestion
  if (sizes.hasOwnProperty(spacing)) {
```

Reply:
good catch!